test/jest.config.server.js

```diff
@@ -22,10 +22,7 @@ export default {
   testPathIgnorePatterns: config.testPathIgnorePatterns.filter(
     (pattern) => !pattern.includes('integration_tests')
   ),
-  setupFilesAfterEnv: [
-    '<rootDir>/src/dev/jest/setup/after_env.integration.js',
-    '<rootDir>/plugins/opendistro_security/test/es/setup_es.js',
-  ],
+  setupFilesAfterEnv: ['<rootDir>/src/dev/jest/setup/after_env.integration.js'],
```

Comment:
Suggest adding a "TODO" here (or inside `after_env.integration.js`) says clean up the data.